sdk/communication/Azure.Communication.CallAutomation/src/Models/CallConnectionProperties.cs

```diff
@@ -19,7 +19,8 @@ public class CallConnectionProperties
             CommunicationIdentifier sourceIdentity,
             PhoneNumberIdentifier sourceCallerIdNumber,
             string sourceDisplayName,
-            string mediaSubscriptionId
+            string mediaSubscriptionId,
+            string dataSubscriptionId
```

Comment:
the word data might not be clear to the developers

Reply:
Will update it later